modules/ROOT/pages/connected-app-bearer-token-example.adoc

```diff
@@ -0,0 +1,58 @@
+= Getting the Bearer Token for a Connected App Example
+
+This example demonstrates how to create a connected app that acts on its own behalf using client credentials to get the bearer token so you can invoke Anypoint Platform APIs. 
+
+[[create-connected-app]]
+== Create a Connected App
+
+To create a connected app, add Runtime Manager scopes, and deploy the app to CloudHub:
+
+include::partial$include-nav-connected-apps.adoc[]
+
+. In the *Owned Apps* section, click *Create App*.
+. In the *Create App* page, enter a unique *Name* for the app, select *App acts on its own behalf (client credentials)*, and then click *Save*.
```

Comment:
```suggestion
. In the *Create App* page, enter a unique name for the app, select *App acts on its own behalf (client credentials)*, and then click *Save*.
```

Reply:
(We [avoid](https://confluence.internal.salesforce.com/display/MTDT/MuleSoft+CX+Writing+Style+Reference#:~:text=Do%20not%20use%20a%20UI%20label%20to%20replace%20normal%20sentence%20syntax%3A) using UI elements this way.